app/views/home/_chat.html.erb

```diff
@@ -1,92 +1,32 @@
 <h1 class="center-heading"><%= I18n.t('dashboard.global_chat') %></h1>
-<div id="chat-messages" class="scroll-list scroll-styled">
-  <div class="chat-item d-flex flex-row">
-    <div class="avatar-container d-flex align-items-end"><a href="#"><span class="w-48 avatar"><i class="fa fa-user"></i></span></a></div>
-    <div class="chat-content">
-      <a href="#" class="chat-author">Sam Kuran</a>
-      <div class="text-sm">
-        For what reason would it be advisable for me to think about business content?
-      </div>
-      <div class="chat-date text-sm d-md-block">
-        24.12.20 13:37
-      </div>
+<div id="chat-messages" class="scroll-list scroll-styled" data-channel-subscribe="room" data-room-id="<%= Room::GLOBAL_CHAT_ID %>">
+  <% @room_messages.each do |room_message| %>
+    <%= render 'room_messages/message',
+               message: room_message %>
+  <% end %>
+</div>
+<%= form_with(model: RoomMessage.new, remote: true) do |f| %>
+  <div class="input-group" id="chat-new-message">
+    <%= f.text_area :message, class: 'form-control',
+                    rows: "2", placeholder: I18n.t('chat.placeholder') %>
+    <%= f.hidden_field :room_id, value: Room::GLOBAL_CHAT_ID %>
+    <div class="input-group-append inputWithIcon">
+      <%= f.submit class: "btn btn-primary", id: "post-chat-button", value: I18n.t('chat.send') %>
```

Comment:
After the submit button is pressed, the text is changed from `Send` to `Create Room message` for a short moment. Depending on the width of the window, the longer text causes the text field and button to be arranged vertically instead of horizontally. Can this behavior be suppressed?

Reply:
Fixed in Commit 4977e552

Reply:
Fixed in 
4977e55